src/ZProc.ml

```diff
@@ -143,7 +210,7 @@ let simplify (z3 : t) (q : string) : string =
   let goal =
     match
       run_queries z3 ~db:["(assert " ^ q ^ ")"]
-                  ["(apply (repeat (then purify-arith simplify ctx-simplify ctx-solver-simplify)))"]
```

Comment:
Did this tactic cause any issues?

Reply:
nope, I think earlier when i was debugging I removed this.

Reply:
I see, then you could revert this change.